src/Symfony/Component/Console/composer.json

```diff
@@ -16,7 +16,8 @@
         }
     ],
     "require": {
-        "php": ">=5.3.9"
+        "php": ">=5.3.9",
+        "symfony/debug": "~2.7,>=2.7.2"
```

Comment:
we don't use `^` yet
we'll add 3.x when merging into 2.8 (the only branch that opens to 3.x)